src/shell/shell.cpp

```diff
@@ -880,7 +891,30 @@ void SHELL_Init() {
 	        "Examples:\n"
 	        "  \033[32;1mset\033[0m\n"
 	        "  \033[32;1mset\033[0m \033[37;1mname\033[0m=\033[36;1mvalue\033[0m\n");
-	MSG_Add("SHELL_CMD_IF_HELP","Performs conditional processing in batch programs.\n");
+	MSG_Add("SHELL_CMD_IF_HELP",
+	        "Performs conditional processing in batch programs.\n");
+	MSG_Add("SHELL_CMD_IF_HELP_LONG",
+	        "Usage:\n"
+	        "  \033[32;1mif\033[0m \033[35;1m[not]\033[0m \033[36;1merrorlevel\033[0m \033[37;1mNUMBER\033[0m COMMAND\n"
+	        "  \033[32;1mif\033[0m \033[35;1m[not]\033[0m \033[37;1mSTR1==STR2\033[0m COMMAND\n"
+	        "  \033[32;1mif\033[0m \033[35;1m[not]\033[0m \033[36;1mexist\033[0m \033[37;1mFILE\033[0m COMMAND\n"
+	        "\n"
+	        "Where:\n"
+	        "  \033[37;1mNUMBER\033[0m     is a positive integer less or equal to the desired value.\n"
+	        "  \033[37;1mSTR1==STR2\033[0m compares two text strings (case sensivitive).\n"
+	        "  \033[37;1mFILE\033[0m       is an exact file name to check for existence.\n"
+	        "  COMMAND    is a DOS command/program to run, optionally with parameters.\n"
+	        "\n"
+	        "Notes:\n"
+	        "  The COMMAND will only run if any of the three conditions is true.\n"
+	        "  If \033[38;1mnot\033[0m is specified, then the command runs only with the false condition.\n"
+	        "  The \033[36;1merrorlevel\033[0m condition is useful for checking if a programs ran correctly.\n"
+	        "  If either \033[37;1mSTR1\033[0m or \033[37;1mSTR2\033[0m may be empty, you can enclose them in quotes (\").\n"
```

Comment:
`"If ... you can enclose them in quotes.. "`

This also reads more like a helpful suggestion, and not a condition. 

 - Do both strings _need_ to exist for the command to be run?
 - Must empty strings be quoted `""` for the command to be run?

Reply:
@kcgen Yes, it was intended to be a suggestion. It is true that both strings need to exist, or else there will be a "syntax error". In such case empty strings must be quoted so that the command will work.